src/App.js

```diff
@@ -251,18 +250,13 @@ function ReleasesMenu({versions}: {versions: OngoingVersions}) {
         style={{borderRight: 'none'}}
       >
         <Menu.SubMenu key="sub1" title="Firefox Releases">
-          <Menu.Item>
-            <a href={localUrlFromVersion(nightly)}>{'Nightly: ' + nightly}</a>
-          </Menu.Item>
-          <Menu.Item>
-            <a href={localUrlFromVersion(beta)}>{'Beta: ' + beta}</a>
-          </Menu.Item>
-          <Menu.Item>
-            <a href={localUrlFromVersion(release)}>{'Release: ' + release}</a>
-          </Menu.Item>
-          <Menu.Item>
-            <a href={localUrlFromVersion(esr)}>{'ESR: ' + esr}</a>
-          </Menu.Item>
+          {versions.map(([channel: string, version: string]) => (
+            <Menu.Item key={channel}>
+              <a
+                href={localUrlFromVersion(version)}
+              >{`${channel}: ${version}`}</a>
```

Comment:
Can we add a capital letter to the channel in the display?

Reply:
How do I deal with "esr"? It would be capitalized to "Esr" which doesn't really make sense, does it?

Should that be sent with the proper casing from Pollbot?